charts/payment-api/values.yaml

```diff
@@ -28,7 +28,7 @@ java:
     FEES_REGISTER_URL: http://fees-register-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     PAYBUBBLE_HOME_URL: https://ccpay-bubble-frontend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CORE_CASE_DATA_API_URL: http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-
+    BULK_SCANNING_URL: http://ccpay-bulkscanning-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
Just a suggestion if it is possible can we have something like this CCPAY_BULK_SCANNING_API_URL

Reply:
I agree. It is a good idea to be more specific with variable names

Reply:
I am happy to make this renaming as per the suggestion